python/dgl/data/lrgb.py

```diff
@@ -0,0 +1,237 @@
+import hashlib
+import os
+import pickle
+
+import pandas as pd
+from ogb.utils import smiles2graph
+from tqdm import tqdm
+
+from .. import backend as F
+
+from ..convert import graph as dgl_graph
+from .dgl_dataset import DGLDataset
+from .utils import download, load_graphs, save_graphs, Subset
+
+
+class PeptidesStructuralDataset(DGLDataset):
+    r"""Peptides structure dataset for the graph classification task.
+
+    DGL dataset of 15,535 small peptides represented as their molecular
+    graph (SMILES) with 11 regression targets derived from the peptide's
+    3D structure.
+
+    The original amino acid sequence representation is provided in
+    'peptide_seq' and the distance between atoms in 'self_dist_matrix' field
+    of the dataset file, but not used here as any part of the input.
```

Comment:
We just provide a dataset interface to the original data. Such details on raw data may not be necessary here.

Reply:
fixed